changelog.d/15041.misc

```diff
@@ -0,0 +1 @@
+Fix a rare exception in logs on start up.
```

Comment:
To check---if this error occurs on startup, will it go away once new device list changes arrive in the stream?

Reply:
I believe so